cli50

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
What should the default shebang be here, @kzidane ?

Reply:
I think that should work given that we're assuming 3.6 and `pip` alters anyway.

Reply:
Right but is there a "correct" way when using `setup.py`? Or does it just overwrite whatever's there?

Reply:
The documentation doesn't specifically mentions a certain preference. Rather it says that if the shebang contains the word `python`, it will adjust to refer to the current interpreter location. So yes, it overwrites whatever's there if it contains `python`.